File: app/src/main/java/com/sec/android/app/shealth/StepWidgetService.java
package com.sec.android.app.shealth;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class StepWidgetService extends RemoteViewsService {
    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new StepRemoteViewsFactory(getApplicationContext());
    }
}

File: app/src/main/java/com/sec/android/app/shealth/StepWidgetService.kt
package com.sec.android.app.shealth

import android.content.Intent
import android.widget.RemoteViewsService

class StepWidgetService : RemoteViewsService() {
    override fun onGetViewFactory(intent: Intent): RemoteViewsFactory {
        return StepRemoteViewsFactory(applicationContext)
    }
}